internal/documentdb/cursor/registry.go

```diff
@@ -135,15 +136,10 @@ func (r *Registry) NewCursor(id int64, continuation wirebson.RawDocument, conn *
 
 	persist := conn != nil
 
-	// TODO https://github.com/FerretDB/FerretDB-DocumentDB/issues/270
+	// TODO https://github.com/FerretDB/FerretDB/issues/5445
 	if len(continuation) == 0 {
-		if persist {
-			r.l.Debug(
-				"Not persisting connection with empty continuation",
-				slog.Int64("id", id), slog.Any("continuation", cont), slog.Bool("persist", persist),
-			)
-
-			_ = conn.Close(context.TODO())
+		if devbuild.Enabled {
+			must.BeZero(conn)
```

Comment:
What I meant is "let's do a somewhat slow decoding under `if devbuild.Enabled {`", not that check